components/director/internal/domain/package/service.go

```diff
@@ -251,8 +255,9 @@ func (s *service) createAPIs(ctx context.Context, packageID, tenant string, apis
 
 		err = s.apiRepo.Create(ctx, api)
 		if err != nil {
-			return errors.Wrap(err, "while creating API for application")
+			return errors.Wrapf(err, "while creating APIDefinition with id %s within Package with id %s", apiDefID, packageID)
 		}
+		log.Infof("Successfully created APIDefinition with id %s within Package with id %s", apiDefID, packageID)
```

Comment:
i think log levels are not consistent in a couple of places - here you log related resource creation on info, while during creating related resources of applications (namely webhooks, the same is logged on debug)

Reply:
My concern was that if many webhooks/labels are included during Application registration, the logs on `Info` will be filled with not that important information which will make them less readable.

From your point of view, is it ok for them to be on `Info`?

Reply:
Maybe we keep labels on debug and hooks on info